src/Illuminate/Auth/Access/Gate.php

```diff
@@ -572,6 +572,8 @@ protected function callBeforeCallbacks($user, $ability, array $arguments)
                 return $result;
             }
         }
+
+        return null;
```

Comment:
Although I also prefer that all `null` returns would be explicit, there are too many places in Laravel that rely on `null` to be implicitly returned.

I'd really like to see this particular change merged, and maybe done for all other places to make everything more explicit, but I wouldn't hold high hopes on that one.